eng/pipelines/templates/jobs/vmr-build.yml

```
@@ -69,6 +69,14 @@ parameters:
   type: string
   default: ''
 
+- name: targetOS
+  type: string
+  default: ''
+
+- name: targetArch
```

nit: let keep the name in sync with the msbuild property:

```suggestion
- name: targetArchitecture
```